apps/backend/src/modules/contributors/contributor.repository.ts

```diff
@@ -56,12 +56,13 @@ class ContributorRepository implements Repository {
 	}
 
 	public async findAll({
+		hasHidden = true,
 		page,
 		pageSize,
-	}: PaginationQueryParameters): Promise<
+	}: { hasHidden?: boolean } & PaginationQueryParameters): Promise<
```

Comment:
Maybe it would be better to create a separate type for this parameters intersection? (and use it in service too)

Reply:
Like it was done with `Project`/`ProjectGroup`/`Notification` `GetAllResponseDto`

Reply:
this parameter is not a part of request dto, but is passed separately from other services. So I would leave it this way